doc/commands.md

```diff
@@ -1222,7 +1222,7 @@ Positional | Description | Type | Notes
 Option | Description | Type | Notes
 ---|---|---|---
 `--bundle` | Create and use a production bundle retaining test hooks | boolean |
-`--cache` | Use HardSourceWebpackPlugin cache | boolean |
+`--cache, --karma.cache` | Enable webpack's caching for test runs | boolean | default: `false`
```

Comment:
Why are there two labels? Is it just because we already have `--coverage` and `--karma.coverage`? Personally, I think aliases just add confusion and would strongly prefer we only have one label per flag, but I'm not gonna die on that hill.